### ISSUE TITLE
Accessibility Fixes

### ISSUE THREAD

**Issue by jessicaoddi** [elementor/elementor#13810]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**

Accessibility errors for icons using an i tag. As well double attribute of "role=navigation" on nav tags which don't require the attribute.

**Describe the solution you'd like**

Replace <i> tags for icons with <span> or another element. Not all icons are for decoration, and this also makes screen readers ignore/miss social link icons.

**Describe alternatives you've considered**

I've replaced my social media links with images of icons in the meantime manually. But it doesn't fix the scroll up icons or other icons I don't want as decoration but function-able elements.

**Additional context**

**Comment by AleksBaikalski** [comment]
+1
Icons in `nav` should have `role="presentation"`. And it would be much more modern and easier to handle approach to use customizable svg bundles (sprites) instead of fonts

**Comment by jessicaoddi** [comment]
[achecker_2021-02-13_03-34-35.pdf](https://github.com/elementor/elementor/files/5975118/achecker_2021-02-13_03-34-35.pdf)

Here's my accessibility report to show what I'm talking about. I built my site (https://jessicaoddi.com) completely with elementor. Another potential error is not having more specific suggestions for form submission error messages.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v3.3.0

Feel free to update

Thanks!

**Comment by jessicaoddi** [comment]
Amazing!



jessica oddi (mobile)

pronounced jEs-i-kuh OH-dee
disabled designer (she/her)

email: ***@***.***
website: jessicaoddi.com
social: @oddi.jessica

•

For accessibility, I am booking clients in advance throughout the year.
Please check in for available time blocks when requesting work. Office
hours are Monday to Friday from 10:30am - 12:00noon and 2:30pm - 6:00pm. I
will respond within 1-2 business days depending on physical ability. Thank
you for understanding!
